tsconfig.json

```diff
@@ -1,18 +1,37 @@
 {
   "compilerOptions": {
-    "baseUrl": "src",
-    "rootDir": "./",
-    "outDir": "build",
-    "target": "esnext",
+    // Basic Options
+    "target": "ES2017",
     "module": "commonjs",
-    "moduleResolution": "node",
+    "declaration": true,
     "sourceMap": true,
-    "removeComments": true,
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+
+    // Directories
+    "outDir": "build",
+
+    // Strict Type-Checking Options
     "strict": true,
-    "resolveJsonModule": true,
-    "esModuleInterop": true,
+    "noImplicitAny": false,
+
+    // Additional Checks
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+
+    // Module Resolution Options
+    "moduleResolution": "node",
+
+    // Advanced Options
     "allowJs": true,
-    "declaration": true,
+    "skipLibCheck": true,
```

Comment:
Why do we `skipLibCheck`?

Does anything break if we remove this line?

Reply:
I added this back when we were doing the weird requires thing with the any's I removed this now good catch.